### ISSUE TITLE
vmwarefusion: failed to load certificate: the certificate is not valid yet

### ISSUE THREAD

**Issue by jerry-x** [kubernetes/minikube#4832]
# minikube start --kubernetes-version v1.13.1
ð  minikube v1.2.0 on darwin (amd64)
✅  using image repository registry.cn-hangzhou.aliyuncs.com/google_containers

⚠️  Ignoring --vm-driver=virtualbox, as the existing "minikube" VM was created using the vmwarefusion driver.
⚠️  To switch drivers, you may create a new VM using `minikube start -p <name> --vm-driver=virtualbox`
⚠️  Alternatively, you may delete the existing VM using `minikube delete -p minikube`

ð  Restarting existing vmwarefusion VM for "minikube" ...
⌛  Waiting for SSH access ...
ð³  Configuring environment for Kubernetes v1.13.1 on Docker 18.09.6
ð  Relaunching Kubernetes v1.13.1 using kubeadm ...

ð£  Error restarting cluster: running cmd: sudo kubeadm init phase certs all --config /var/lib/kubeadm.yaml: command failed: sudo kubeadm init phase certs all --config /var/lib/kubeadm.yaml
stdout: [certs] Using certificateDir folder "/var/lib/minikube/certs/"

stderr: error execution phase certs/front-proxy-ca: failure loading front-proxy-ca certificate: failed to load certificate: the certificate is not valid yet
: Process exited with status 1

ð¿  Sorry that minikube crashed. If this was unexpected, we would love to hear from you:
ð  https://github.com/kubernetes/minikube/issues/new
❌  Problems detected in "kube-addon-manager":
    error: no objects passed gt deofa ualptp ly

when I've finished installing minikube on macos using vmwarefusion as the vm driver,everyting's right,but when I stopped the minikube and started it again,it reported this error above,anyone knows  how to  resolve it?

**Comment by hugomcruz** [comment]
I have exactly the same issue just after installation. I also use vmwarefusion on mac as the driver. It started perfectly, then I stopped and then I tried to start I had the error above. 
The only way to make it work is deleting the config: "minikube delete" and then start again.

**Comment by tstromberg** [comment]
Do you mind including the output of:

` minikube start --vm-driver=vmwarefusion -v=7 --alsologtostderr`

minikube should synchronize time when reusing a previously created VM. It at least appears to in minikube v1.3.0 with this driver:

```
Setting Docker configuration on the remote daemon...
I0808 14:15:41.862422   23097 cluster.go:204] guest clock: 1565298941.862183450
I0808 14:15:41.862450   23097 cluster.go:217] Guest: 2019-08-08 14:15:41.86218345 -0700 PDT Remote: 2019-08-08 14:15:41.638209 -0700 PDT m=+2.819088089 (delta=223.97445ms)
I0808 14:15:41.862481   23097 cluster.go:188] guest clock delta is within tolerance: 223.97445ms
I0808 14:15:42.025024   23097 start.go:922] Saving config:
```

**Comment by hugomcruz** [comment]
After running the command line you sent it actually worked. 
Even if I do  "minikube start" it now works. I am not sure exactly why, but it works. 

Thanks for the help.

**Comment by tstromberg** [comment]
@jerry-x: I believe this issue is now addressed by minikube v1.4, as it now forcibly syncs the time across all starts. If you still see this issue with minikube v1.4 or higher, please reopen this issue by commenting with `/reopen`.

Also worth noting: the `vmwarefusion` driver is deprecated, and was replaced with `vmware`: https://minikube.sigs.k8s.io/docs/reference/drivers/vmware/

Thank you for reporting this issue!
